packages/accounts-base/accounts_client_tests.js

```diff
@@ -178,6 +192,37 @@ Tinytest.addAsync(
   }
 );
 
+Tinytest.addAsync(
+  'accounts async - Meteor.userAsync() obeys explicit and default field selectors',
+  (test, done) => {
+    logoutAndCreateUser(test, done, () => {
+      Meteor.loginWithPassword(username, password, () => {
+        // by default, all fields should be returned
+        Meteor.userAsync().then(user => {
```

Comment:
It'd be good to `await` all of these promises, as right now there's no way of making sure the test actually passes.